### PR TITLE
Reference ErrorInterface, not Error

### PR DESCRIPTION
**Pull request by lindyhopchris**
As per #141

### CODE REVIEW

File: src/Contracts/Http/ResponsesInterface.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-use \Neomerx\JsonApi\Document\Error;
+use \Neomerx\JsonApi\Contracts\Document\ErrorInterface;
```
Comment (neomerx) [inline src/Contracts/Http/ResponsesInterface.php]: Can you please move this line down?

use \aaaaaa;
use \bbbbbbbb;
use \ccccccccccccc;

File: src/Contracts/Http/ResponsesInterface.php [hunk 2]
```diff
@@ -94,7 +94,7 @@ public function getMetaResponse($meta, $statusCode = self::HTTP_OK, array $heade
     /**
      * Get response with JSON API Error in body.
      *
-     * @param Error|Error[]|ErrorCollection $errors
+     * @param ErrorInterface|ErrorInterface[]|ErrorCollection $errors
      * @param int                           $statusCode
      * @param array                         $headers
```
Comment (neomerx) [inline src/Contracts/Http/ResponsesInterface.php]: alignment broken

File: src/Exceptions/ErrorCollection.php [hunk 3]
```diff
@@ -21,9 +21,10 @@
 use \ArrayObject;
 use \Serializable;
 use \IteratorAggregate;
-use \Neomerx\JsonApi\Document\Error;
-use \Neomerx\JsonApi\Contracts\Document\LinkInterface;
 use \Neomerx\JsonApi\Contracts\Document\DocumentInterface;
+use \Neomerx\JsonApi\Contracts\Document\ErrorInterface;
+use \Neomerx\JsonApi\Contracts\Document\LinkInterface;
+use \Neomerx\JsonApi\Document\Error;
```
Comment (neomerx) [inline src/Exceptions/ErrorCollection.php:24]: same as above (line order)

File: src/Exceptions/JsonApiException.php [hunk 4]
```diff
@@ -18,7 +18,7 @@
 
 use \Exception;
 use \RuntimeException;
-use \Neomerx\JsonApi\Document\Error;
+use \Neomerx\JsonApi\Contracts\Document\ErrorInterface;
```
Comment (neomerx) [inline src/Exceptions/JsonApiException.php]: line order

File: src/Exceptions/JsonApiException.php [hunk 5]
```diff
@@ -57,7 +57,7 @@ class JsonApiException extends RuntimeException
     /**
      * Constructor.
      *
-     * @param Error|Error[]|ErrorCollection $errors
+     * @param ErrorInterface|ErrorInterface[]|ErrorCollection $errors
```
Comment (neomerx) [inline src/Exceptions/JsonApiException.php:60]: alignment

File: src/Http/Responses.php [hunk 6]
```diff
@@ -16,15 +16,15 @@
  * limitations under the License.
  */
 
-use \Neomerx\JsonApi\Document\Error;
-use \Neomerx\JsonApi\Exceptions\ErrorCollection;
-use \Neomerx\JsonApi\Contracts\Http\ResponsesInterface;
+use \Neomerx\JsonApi\Contracts\Document\ErrorInterface;
 use \Neomerx\JsonApi\Contracts\Encoder\EncoderInterface;
-use \Neomerx\JsonApi\Contracts\Schema\ContainerInterface;
+use \Neomerx\JsonApi\Contracts\Encoder\Parameters\EncodingParametersInterface;
```
Comment (neomerx) [inline src/Http/Responses.php]: line order

**Comment by neomerx** [comment]
Can you re-send fixed PR to `develop` with comment 'Close #141'?

**Comment by neomerx** [comment]
Oh! We have a special branch for it! Nice. I can fix it myself.

**Comment by neomerx** [comment]
@lindyhopchris I've made a tiny code style changes. Can you please merge it into 1 commit with comment 'Close #141' and merge into 'develop'?

File: src/Contracts/Http/ResponsesInterface.php (hunk 1 above)
Comment (lindyhopchris) [inline src/Contracts/Http/ResponsesInterface.php]: Yeah, this is different from how PHP Storm orders the use statements. I'm also not used to having to put `\` at the start of a `use` statement.

**Pull request closed without merging**